### PR TITLE
Actually upgrade mathjax-renderer to the version specified in the catalog

### PR DESCRIPTION
**Pull request by benchristel**
## Summary:
In a previous PR, I upgraded mathjax-renderer in the catalog, but forgot to
pnpm install.  This wasn't causing CI failures, I'm guessing because we install
from a frozen lockfile in CI, but typechecking was failing locally after I ran
`pnpm i`.

Issue: none

## Test plan:

```
pnpm tsc -w
```

### CODE REVIEW

File: testing/test-mathjax.tsx [hunk 1]
```diff
@@ -45,3 +47,44 @@ export function TestMathjax({children: tex, onRender}: Props) {
 
     return <span ref={ref} />;
 }
+
+function isPresentationalForSR(element: HTMLElement | null): boolean {
+    if (element == null) {
+        return false;
+    }
+    return (
+        presentationalRoles.some((role) => element.matches(`[role=${role}]`)) ||
```
[the remark this answers is not in the document]
Comment (catandthemachines) [reply testing/test-mathjax.tsx:60]: That makes sense, did you model this new logic off of the previous logic of `addLabelWhenPresentational`? I'm curious because the creation of the const `presentationalRoles` gives me pause or weird vibes for hard coding the different roles that can be used under `role=presentation`.

What was the intension behind that addition?

File: testing/test-mathjax.tsx [hunk 2]
```diff
@@ -25,18 +25,20 @@ type Props = {
...
         () => renderer.render(tex),
         [tex],
     );
 
     React.useLayoutEffect(() => {
         if (ref.current) {
-            addLabelWhenPresentational(ref.current);
+            if (isPresentationalForSR(ref.current)) {
+                addLabel().catch(console.error);
+            }
```
[the remark this answers is not in the document]
Comment (jeremywiebe) [reply testing/test-mathjax.tsx:41]: 🤔 I sometimes wonder if injecting a TeX component is just wrong and if Perseus should own that. We used to have multiple math rendering libraries in use and more complexity, but perhaps in 2025 we could just let Perseus use `mathjax-renderer` directly and eliminate the `TeX` dependency.

**Review by jeremywiebe — COMMENTED** [review]
I agree this is the right fix for now. Glad we're queuing up a ticket for the future. Thanks Ben.

**Review by catandthemachines — APPROVED** [review]
Approve as long as we have a LEMS ticket to revisit the loss of `addLabelWhenPresentational` in the future ❤️
